caom2utils/caom2utils/tests/test_fits2caom2.py

```diff
@@ -94,7 +94,7 @@
 from lxml import etree
 
 from unittest.mock import Mock, patch
-from six import StringIO, BytesIO
+from io import StringIO, BytesIO
```

Comment:
The first `import` is `import io`. Since this import exists, I think it can be deleted.